doc/manual/types.rst

```diff
@@ -962,7 +962,7 @@ immutable type::
 
 However, there are three key differences:
 
-- Tuple types may have any number of parameters.
+- Tuple types may have any number of parameters, including 1.
```

Comment:
IMO that doesn't really make it clearer; I would assume that "any number of parameters" includes 1. If anything I would say "including 0," since that may be less obvious.